### PR TITLE
Experiment to test server hello content type

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: addons/tls13-middlebox-serverhello/bootstrap.js [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+function makeRequest(config) {
+  return new Promise((resolve, reject) => {
+    // put together the configuration and the info collected from the connection
+    async function reportResult(event, xhr) {
+      resolve(Object.assign({"event": event, "responseCode": xhr.status}, await getInfo(xhr)));
+      return true;
+    }
+
+    try {
+      let xhr = Cc["@mozilla.org/xmlextras/xmlhttprequest;1"].createInstance(Ci.nsIXMLHttpRequest);
```
Comment (rhelmer) [inline addons/tls13-middlebox-serverhello/bootstrap.js]: Not important but I think you can do the less-wordy `Cu.importGlobalProperties(["XMLHttpRequest"])` instead if you want.

**Comment by rhelmer** [comment]
BTW looks like the failure is just this minor eslint thing:
```
  /home/worker/repo/addons/tls13-middlebox-serverhello/bootstrap.js
  34:113  error  Extra space after key 'tlsFlags'  key-spacing
```

File: addons/tls13-middlebox-serverhello/bootstrap.js (hunk 1 above)
Comment (ekr) [reply addons/tls13-middlebox-serverhello/bootstrap.js]: In the interest of laziness, I'm going to leave it as-is :)

**Comment by ekr** [comment]
@rhelmer: can you merge this?

**Pull request merged**